invenio_rdm_records/services/services.py

```diff
@@ -295,3 +295,15 @@ def delete_secret_link(
         self._index_related_records(record, parent)
 
         return True
+
+    def reseve_doi(
+        self,
```

Comment:
They can all go in one line, but that's my personal preference no need to do it 😅